src/common/Button.jsx

```diff
@@ -38,7 +39,7 @@ Button.propTypes = {
   name: PropTypes.string.isRequired,
   className: PropTypes.string.isRequired,
   value: PropTypes.string.isRequired,
-  onClick: PropTypes.func.isRequired,
+  onClick: PropTypes.func,
```

Comment:
Shouldn't we require anyone using this component to provide a click handler since it is a button?

Reply:
Good spot when I changed this when I was getting the bug on the buttons that were breaking. I will change this back